local/bin/py/build/pull_rbac.py

```diff
@@ -31,18 +41,42 @@ def pull_rbac():
             print(f'\x1b[33mWARNING\x1b[0m: RBAC api request failure reason: {e}')
             pass
 
-    if json_result:
-        data = json_result.get('data', [])
+    if permissions_json and roles_json:
+        permissions_data = permissions_json.get('data', [])
+        # gets default roles (admin, standard, and read-only)
+        # ignores UI template roles ('Datadog Billing Admin Role' and 'Datadog Security Admin Role')
+        default_roles_data = roles_json.get('data', [])[0:3] 
```

Comment:
is there any chance this data could come back in a different order? or that there isn't 3 items? We might want to extract this another way